README.md

```diff
@@ -16,7 +16,7 @@ Includes support for:
 
 1) Download and install Steampipe (https://steampipe.io/downloads).
 
-2) Install the Azure plugin with [Steampipe](https://steampipe.io):
+2) Install the Azure and the Azure Active Directory plugins with [Steampipe](https://steampipe.io):
```

Comment:
We mention the Azure AD plugin, but didn't update the instructions below

Reply:
Similarly, below, if I just run `az login`, does that work for both the `azure` and `azuread` plugins? Just curious since the Azure plugin uses the resource manager/global endpoint, while Azure AD uses the Graph endpoint.